$MirOS: ports/www/dillo/v2/patches/patch-dpi_bookmarks_c,v 1.3 2010/02/21 20:08:42 bsiegert Exp $
--- dpi/bookmarks.c.orig	Sat Jun 27 08:29:58 2009
+++ dpi/bookmarks.c	Sun Aug 23 17:03:44 2009
@@ -691,7 +691,7 @@ static void Bms_check_import(void)
 
 
    if (access(BmFile, F_OK) != 0) {
-      OldBmFile = dStrconcat(dGethomedir(), "/.dillo/bookmarks.html", NULL);
+      OldBmFile = dStrconcat(dGethomedir(), "/.etc/dillo/bookmarks.html", NULL);
       if (access(OldBmFile, F_OK) == 0) {
          dStr_sprintf(dstr, cmd1, BmFile);
          rc = system(dstr->str);
@@ -1708,7 +1708,7 @@ int main (void) {
    /* Initialize local data */
    B_bms = dList_new(512);
    B_secs = dList_new(32);
-   BmFile = dStrconcat(dGethomedir(), "/.dillo/bm.txt", NULL);
+   BmFile = dStrconcat(dGethomedir(), "/.etc/dillo/bm.txt", NULL);
    /* some OSes may need this... */
    address_size = sizeof(struct sockaddr_un);
 
